### ISSUE TITLE
The Find window does not close on the first Esc

### ISSUE THREAD

**Issue by yozhic** [Alexey-T/CudaLister#105]
В 32-битном плагине окно `Find` (вызванное по <kbd>F3</kbd>) не закрывается по 1му нажатию <kbd>Esc</kbd>. Мигает и остается открытым. Для закрытия нужно нажать <kbd>Esc</kbd> 2й раз.

В 64-битном плагине этого нет. Здесь окно `Find` закрывается по 1му нажатию <kbd>Esc</kbd>.

Конфигурация:
Windows 10 x64 (21H2 Enterprise, build 19044); Total Commander 10.50 combo; CudaLister 1.8.0.1

**Comment by springsunx** [comment]
In 32-bit, many operations will be repeated. Ctrl+V will paste twice, and F3 will skip the second one and find the third one.
I have found these problems, but have not found a suitable solution.

**Comment by yozhic** [comment]
> Ctrl+V will paste twice

Confirmed. And <kbd>Ctrl+Z</kbd> will Undo twice.

> F3 will skip the second one and find the third one

Confirmed/

**Comment by yozhic** [comment]
Yes, tested on Windows XP SP3 x86 (TC 10.52, CudaLister 1.8.0.1): exactly the same behavior.

**Comment by Alexey-T** [comment]
то же самое что #113. только что правил. выпустил 1.8.5.2. решено там?
